Readers/src/TriggerInfoReader.cc

```diff
@@ -40,12 +89,43 @@ TriggerInfoReader::setBranchAddresses(TTree * tree)
       }
     }
   }
+  bai.setBranchAddress(ntriggerObj_, branchName_num_);
+  bai.setLenVar(64).setBranchAddress(triggerObj_id_, branchName_triggerObj_id_);
```

Comment:
Where's the number `64` coming from? Can you make it const member or static variable of the class?